tests/integration/test_sca/test_scan_results.py

```diff
@@ -0,0 +1,141 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: These tests will that a scan is ran using the configured sci_sca ruleset and regex engine.
+
+components:
+    - sca
+
+suite: sca
+
+targets:
+    - manager
+    - agent
+
+daemons:
+    - wazuh-modulesd
+
+os_platform:
+    - linux
+
+os_version:
+    - CentOS 8
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/index.html
+
+tags:
+    - sca
+'''
+import os
+import pytest
+
+from wazuh_testing import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.modules.sca import event_monitor as evm
+from wazuh_testing.modules.sca import SCA_DEFAULT_LOCAL_INTERNAL_OPTIONS as local_internal_options
+
+
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0)]
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_enabled.yaml')
+cases_path = os.path.join(TEST_CASES_PATH, 'cases_scan_results.yaml')
+
+# Test configurations
+configuration_parameters, configuration_metadata, case_ids = get_test_cases_data(cases_path)
+configurations = load_configuration_template(configurations_path, configuration_parameters, configuration_metadata)
+
+
+# Tests
+@pytest.mark.parametrize('configuration, metadata', zip(configurations, configuration_metadata), ids=case_ids)
+def test_scan_results(configuration, metadata, prepare_cis_policies_file, truncate_monitored_files,
```

Comment:
Could you change the test name to anyone more representative, please?

Reply:
done in https://github.com/wazuh/wazuh-qa/pull/3566/commits/d25d892f4afe354447acdf538c6e63f23bcf0614